model/patch_lifecycle.go

```diff
@@ -659,136 +659,90 @@ func RetryCommitQueueItems(projectID string, patchType string, opts RestartOptio
 		}
 		return toBeRequeued, nil, nil
 	}
-	if patchType == commitqueue.SourcePullRequest {
-		restarted, notRestarted := restartPRItems(patches, cq)
-		return restarted, notRestarted, nil
-	}
-	if patchType == commitqueue.SourceCommandLine {
-		restarted, notRestarted := restartCLIItems(patches, cq)
-		return restarted, notRestarted, nil
-	}
-	return nil, nil, errors.Errorf("patch type '%s' is invalid", patchType)
-}
-
-func restartPRItems(patches []patch.Patch, cq *commitqueue.CommitQueue) ([]string, []string) {
-	restartedPatches := []string{}
-	patchesWithErrors := []string{}
+	patchesRestarted := []string{}
+	patchesFailed := []string{}
 	for _, p := range patches {
-		// reconstruct commit queue item from patch
-		modules := []commitqueue.Module{}
-		for _, modulePatch := range p.Patches {
-			if modulePatch.ModuleName != "" {
-				module := commitqueue.Module{
-					Module: modulePatch.ModuleName,
-					Issue:  modulePatch.PatchSet.Patch,
-				}
-				modules = append(modules, module)
-			}
-		}
-		item := commitqueue.CommitQueueItem{
-			Issue:   strconv.Itoa(p.GithubPatchData.PRNumber),
-			Modules: modules,
-			Source:  commitqueue.SourcePullRequest,
+		// use the PR number to determine if this is a PR or diff patch. Currently there
+		// is not a reliable field that is set for CLI patches
+		var err error
+		if p.GithubPatchData.PRNumber > 0 {
+			err = restartPRItem(p, cq)
+		} else {
+			err = restartCLIItem(p, cq)
```

Comment:
I think we should also rename this to diff, for consistency

Reply:
done